pkg/kubefnord/join.go

```diff
@@ -401,30 +402,32 @@ func createRBACSecret(hostClusterClientset, joiningClusterClientset client.Inter
 	namespace, joiningClusterName, hostClusterName,
 	secretName string, dryRun bool) (*corev1.Secret, error) {
 
-	glog.V(2).Info("Creating service account in joining cluster")
+	glog.V(2).Info("Creating service account in joining cluster: %s", joiningClusterName)
 
 	saName, err := createServiceAccount(joiningClusterClientset, namespace,
 		joiningClusterName, hostClusterName, dryRun)
 	if err != nil {
-		glog.V(2).Infof("Error creating service account in joining cluster: %v", err)
+		glog.V(2).Infof("Error creating service account %s in joining cluster: %v",
+			saName, err)
```

Comment:
Why not use the `joiningClusterName` also here to be consistent with other prints.

Reply:
Done